community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
+Generally, registration can be reimbursed if the attendee is speaking on behalf of OpenJS, the CPC, or OpenJS-hosted projects.
+Funding is not transferable to another person, or any other event. 
+
+Travel fund assistance can only be received once per year.
+
+## How To Apply
+
+### Submit Travel Fund Request
+
+Submit your travel fund request to the Cross Project Council by completing [this application](https://forms.gle/QDt3iqoXXB5Ycovz8).
+Please see [Deadlines and Timelines](#deadlines-and-timelines) for information about cut-off dates. 
+
+### Wait For Approval
+
+The Cross Project Council reviews requests once per month. If your request is approved, you will be notified via email.
+
+## Getting Reimbursed
+
+### Submitting Expenses
+
+After the event, kindly submit your expenses within 30 days:
+
+* Fill out these [two forms](https://drive.google.com/drive/folders/1E-dTuqnIWpZN2NP-1ioAWzK_W2zSwa_P?usp=share_link)
+* Submit the forms along with **all** receipts [here](https://form.asana.com/?k=S6lGzAjHv2uv7M8llnhO_w&d=9283783873717)
+
+Please note that expenses without receipts can not be paid.
+
+### Submitting A Trip Report
+
+The process for submitting a trip report is TBD.
+<!-- Provide a trip report by filling out the following form. Preferably, also provide pictures and video of the 
+event for use on social media. -->
+
+### Receiving Payment
+
+Payments are processed within 30 days.
+
+## Deadlines and Timelines 
+
+The default deadline for submitting a community fund request is 8 weeks prior to the travel date to allow for all requests for an event to be reviewed at the same time. 
+
+Some events might have a fixed deadline for applications. Those deadlines will be posted to this repository.
+
+## CPC Review Schedule & Process 
+
+- CPC will review requests once a month in the CPC private session
+- Pending requests will be sent to the private CPC mailing list prior to the review meeting
+- Initial approval will be made by simple majority of the voting and regular members in attendance
+- Non-present members will have the oppurtunity to object to a decision over email
```

Comment:
```suggestion
- Approval will be made through the [consensus seeking process](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#consensus-seeking-process).
```
There's no reasons, imho, to introduce a new decision-making mechanism here.

Reply:
I looked at what we have in the Node.js project and it has the link to the wiki page but nothing else. My key point in the in the recent working group meeting is that the "consensus" must represent that of the CPC, not just the subset of CPC members who could make a particular meeting. That requires that people not in attendance have the opportunity to object before the decision is considered final. I think we might want to update the description in https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#consensus-seeking-process to reflect that as well. For example, if two people show up at a CPC meeting they are not able to make a decision on behalf of all of the CPC.  Further even if 51% of the people show up at a meeting the other 49% of the members should still have an opportunity to object before the decision is considered final.

Reply:
@mhdawson: I agree 100% on both needing to tighten our definition of consensus and on doing it in the right place (i.e. not here). Should we open an issue for this?

Reply:
Yes please spin this off into another issue so that we can unblock merging this pr.

Reply:
I spun this out into it's own issue and will remove the text from this PR pending the CPC's decision.